Misc/NEWS.d/next/IDLE/2019-08-07-23-27-54.bpo-21261.sqNQCD.rst

```diff
@@ -0,0 +1 @@
+IDLE's shell auto-completion completes dict keys of type str and bytes.
```

Comment:
The main suggestion here is adjusting "auto-completion completes" -> "auto-completion works for" and "type" -> "types". 

The inline markup for str and bytes is a minor extra addition, sphinx converts this into a link. Respectively: [str](https://docs.python.org/3.9/library/stdtypes.html#str) and [bytes](https://docs.python.org/3.9/library/stdtypes.html#bytes). See [common sequence operators](https://docs.python.org/3.9/library/stdtypes.html#common-sequence-operations), in the section for the first note. Definitely feel free to remove or modify this part if you don't want it included.

```suggestion
IDLE's shell auto-completion works for dict keys of types :class:`str` and :class:`bytes`.
```

Reply:
Great suggestions, thanks!